content/posts/newsletter-011/index.md

```diff
@@ -432,6 +432,65 @@ written with the framework on platforms without threading - notably, web.
 [`hecs`]: https://crates.io/crates/hecs
 [Rayon]: https://crates.io/crates/rayon
 
+### [nanoserde]
```

Comment:
I love the work done here, but I'm not sure if featuring this in the newsletter is desired. The crate has technically nothing to do with gamedev, if I understand correctly.

Reply:
It is a thin line between gamedev-related and non-gamedev libraries. 
Technically sound engines, networking engines and scripting languages are not gamedev-specific as well. 
Nanoserde was made with games/asset pipelines in mind and the demo is about tiled json parsing.
 
However I do not really mind removing nanoserde from the newsletter, I agree that this is not 100% gamedev library, up to the moderation team or whoeve is responsible for the newsletter moderation.

Reply:
An argument for keeping nanoserde in the newsletter: it may be useful to fellow gamedevelopers who missed the reddit post. 
Maybe we have some bonus/offtopic section for this kind of stuff?

Reply:
some things are *clearly* gamedev related more than with other uses. For example, ECS stuff.
But serialization is super generic and universally useful, there is nothing to it that is better for gamedev than to any other area.

Reply:
Quick compilation is one of *quad ecosystem's UPSs - what about converting nanoserde into macroquad's subsection?

Reply:
https://github.com/rust-gamedev/rust-gamedev.github.io/pull/200/commits/c5ccb8c07f57b19a3b0ae066b93bfb556530e579